app/adapters/external/cache/infinite_buying_adapter.py

```
@@ -211,14 +225,16 @@ async def get_state_with_rounds(self, market: str) -> InfiniteBuyingState | None
             state = await self.get_state(market)
             if state is None:
                 return None
-                
+
             # 매수 회차 정보도 함께 로드
             rounds = await self.get_buying_rounds(market)
             state.buying_rounds = rounds
-            
+
             return state
         except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)